src/ResourceManager/Network/Commands.Network/help/New-AzureRmDelegation.md

```diff
@@ -0,0 +1,103 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermdelegation
+schema: 2.0.0
+---
+
+# New-AzureRmDelegation
```

Comment:
why use this if [Add](https://github.com/Azure/azure-powershell/pull/7220/files#diff-60d47c921815fbe400528e2a479f1558R8) already exists and can do pretty much the same thing?

Reply:
see response to other comment - this can be used in conjunction with new Subnet operations.
Why not add the flexibility to allow customers to use the approach they like best?